app/openlxp_xds_project/settings.py

```diff
@@ -44,6 +44,8 @@
     'rest_framework',
     'rest_framework.authtoken',
     'openlxp_notifications',
+    'social_django',
```

Comment:
Is it necessary to specify social_django here?  I thought that maybe it would be encapsulated within the openlxp_authentication reference.

Reply:
social_django needs to be here so that Django will apply the social_django migrations to the database